test/e2e/flask/multichain-api/get-session.spec.ts

```diff
@@ -0,0 +1,80 @@
+import _ from 'lodash';
+import { withFixtures } from '../../helpers';
+import { Driver } from '../../webdriver/driver';
+import FixtureBuilder from '../../fixture-builder';
+import { DEFAULT_FIXTURE_ACCOUNT } from '../../constants';
+import {
+  DEFAULT_OPTIONS,
+  getExpectedSessionScope,
+  getSessionScopes,
+  openDappAndConnectWallet,
+} from './testHelpers';
+
+describe('Multichain API', function () {
+  describe('Connect wallet to the multichain dapp via `externally_connectable`, call `wallet_getSession` when there is no existing session', function () {
+    it('should successfully receive empty session scopes', async function () {
+      await withFixtures(
+        {
+          title: this.test?.fullTitle(),
+          fixtures: new FixtureBuilder().withPopularNetworks().build(),
+          ...DEFAULT_OPTIONS,
+        },
+        async ({
+          driver,
+          extensionId,
+        }: {
+          driver: Driver;
+          extensionId: string;
+        }) => {
+          await openDappAndConnectWallet(driver, extensionId);
+          const parsedResult = await getSessionScopes(driver);
+
+          if (!_.isEqual(parsedResult.sessionScopes, {})) {
+            throw new Error('Should receive empty session scopes');
```

Comment:
same comment about `assert`

Reply:
yes was about to comment on this

Reply:
resolved in [396ec6f](https://github.com/MetaMask/metamask-extension/pull/29298/commits/396ec6ffcf728182695a52a8f42b9c4b72122933)